packages/Tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+
+import cx from 'classnames';
+
+import CrownSolid from '@igloo-ui/icons/dist/CrownSolid';
+import TextBulletSolid from '@igloo-ui/icons/dist/TextBulletSolid';
+
+import './tabs.scss';
+
+export interface TabInterface {
+  /** The tab pane contents of the tabs list */
+  children?: React.ReactNode;
+  /** Add an id attribute to the tab */
+  id?: string;
+  /** Add text for the tab */
+  label: React.ReactNode;
+  /** Add a notification indicator to the tab */
+  notification?: boolean;
+  /** Add a premium indicator to the tab */
+  premium?: boolean;
+}
+
+export interface TabsProps extends React.ComponentProps<'div'> {
+  /** The function that's called when a tab is clicked */
+  onSelectTab?: (index: number) => void;
+  /** The index number of the Tab that should be selected on load */
+  selected?: number;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
+  /** Add a specific class to the Tabs component */
+  className?: string;
+  /** Add an inline style to the Tabs */
+  isInline?: boolean;
+  /** An array of objects that specify the props of the tab buttons */
+  tabs: Array<TabInterface>;
+}
+
+const Tabs: React.FunctionComponent<TabsProps> = ({
+  onSelectTab,
+  selected = 0,
+  dataTest,
+  className,
+  isInline = true,
+  tabs,
+}) => {
+  const classes = cx('ids-tabs', className, {
+    'ids-tabs--inline': isInline,
+    'ids-tabs--heading': !isInline,
+  });
+
+  /* Handle the on click */
+  const handleOnClick = (index: number): void => {
+    if (onSelectTab) {
+      onSelectTab(index);
+    }
+  };
+
+  /* Render the Tab */
```

Comment:
I don't think we need a comment here, the constant name already define what it is good enough.